styleguide/component-wrapper.js

```diff
@@ -2,13 +2,17 @@ import hljs from 'highlight.js'; // eslint-disable-line
 import beautify from 'js-beautify'; // eslint-disable-line
 
 const a11yid = 'a11yComponentToTest';
-const wrapper = (title, component, usage) => {
+const wrapper = (title, component, usage, js) => {
     const source = hljs.highlightAuto(
         beautify.html_beautify(component, {
             indent_size: 2
         })
     ).value;
 
+    if (js && js) {
```

Comment:
Not sure I understand what's happening here...

Reply:
left over from a revision where I'd used an object - cheers - I'll clean it up.